packages/kotti-ui/source/kotti-popover/KtPopover.vue

```diff
@@ -19,13 +26,27 @@
 import { createPopper } from '@popperjs/core'
 import { mixin as clickaway } from 'vue-clickaway'
 
+import KtPopoverIconTextItem from './KtPopoverIconTextItem.vue'
+
+const optionIsValid = (option) =>
```

Comment:
i noticed you want to support `icon` on the `option` as well but you don't check for it in the validator

check if an icon is passed (not `undefined`) and if it is, there's a util function `isYocoIcon` that you can run immediately